test/functional/json/jsonb.ts

```diff
@@ -35,19 +35,17 @@ describe("jsonb type", () => {
     })));
 
     it("should persist jsonb string correctly", () => Promise.all(connections.map(async connection => {
-        await connection.syncSchema(true);
         let recordRepo = connection.getRepository(Record);
         let record = new Record();
-        record.data = JSON.stringify({ foo: "bar" });
+        record.data = "foo";
```

Comment:
wait, why are you storing string here? Isnt jsonb supposed to support json objects? Can't you do `record.data = { a: 1 };` then after you load it `foundRecord.data === { a: 1 }` ?